alws/crud/sign_task.py

```diff
@@ -402,6 +407,15 @@ async def complete_gen_key_task(
         payload.file_name,
     )
     roles = [r for r in gen_key_task.product.team.roles if 'signer' in r.name]
+
+    await db.execute(
+        update(models.SignKey).where(
+            and_(models.SignKey.product_id == gen_key_task.product.id,
+                 models.SignKey.active)
+        ).values(active=False, archived=datetime.datetime.utcnow())
+    )
+    await db.commit()
```

Comment:
If I'm not mistaken, calling the `commit` method will close the current session and all actions related to the closed session after that will throw an exception

Reply:
I don't believe this is the case, I didn't run into such an error when testing. Additionally, the docs don't seem to support this: https://docs.sqlalchemy.org/en/20/orm/session_api.html#sqlalchemy.orm.Session.commit 
However I may be mistaken here and would defer to others. @javihernandez ?

Reply:
It's fine if it works, probably I mixed it up with something different case or it was a case for sqlalchemy 1.4